manifest_template.yml

```diff
@@ -44,81 +53,84 @@ apigee:
 
 {% for ENV in APIGEE_ENVIRONMENTS %}
 {% set portal_visibility = ENV.portal_visibility | default(true) %}
-{% set TITLE = 'e-Referrals-Service (' + ENV.display_name + ')' %}
-{% set DESCRIPTION = 'e-Referrals-Service - ' + ENV.display_name | lower + ' environment' %}
+{% set TITLE = 'e-Referral Service' %}
 {% set NAME = 'e-referrals-service-api-' + ENV.name %}
 {% if ENV.name == 'internal-dev' %}
   - name: {{ ENV.name }}
     products:
 {% for VARIANT in INTERNAL_DEV_VARIANTS %}
-      - name: e-referrals-service-api-{{ VARIANT }}-{{ ENV.name }}
+{% for MODE in ACCESS_MODES %}
+      - name: e-referrals-service-api-{{ VARIANT }}-{{ ENV.name }}{{ MODE.nameSuffix }}
         approvalType: {{ ENV.approval_type | default('auto') }}
         attributes:
           - name: access
             value: public
           - name: ratelimit
             value: {{ ENV.ratelimit | default('5ps') }}
-        description: {{ DESCRIPTION }} ({{ VARIANT }})
-        displayName: {{ TITLE }} {{ VARIANT }}
+        description: {{ MODE.description }}
+        displayName: {{ TITLE }} - {{ MODE.displayName }} ( {{ ENV.display_name}} - {{ VARIANT }} )
         environments: [ {{ ENV.name }} ]
         proxies:
           - e-referrals-service-api-{{ VARIANT }}-{{ ENV.name }}
           - identity-service-{{ ENV.name }}
-        scopes:
-          - 'urn:nhsd:apim:app:level3:e-referrals-service-api'
-          - 'urn:nhsd:apim:user-nhs-id:aal3:e-referrals-service-api'
+        scopes: {{ MODE.scopes }}
         quota: {{ ENV.quota | default('300') }}
         quotaInterval: '1'
         quotaTimeUnit: minute
+{% endfor %}
 {% endfor %}
     specs:
       - name: {{ NAME }}
         path: e-referrals-service-api.json
     api_catalog:
 {% for VARIANT in INTERNAL_DEV_VARIANTS %}
-      - edgeAPIProductName: e-referrals-service-api-{{ VARIANT }}-{{ ENV.name }}
+{% for MODE in ACCESS_MODES %}
```

Comment:
Just looking at the previous logic here compared to the specs that are deployed in Apigee currently and was wondering why we don't have three specs for the internal-dev environment (one per variant) and instead just have one spec for internal-dev. With this change I'd still expect us to have three specs in the internal-dev environment (as we only currently have one access mode) with that increasing to six within the introduction of a new access mode, but I guess given the existing behaviour this might not be the case? Just thinking it's worth understanding this given we're making an alteration here.

Reply:
Possibly due to the way it uses {{NAME}} to set the {{specId}}? Which would just be internal-dev for all of the internal-dev envs.

I don't know if the {{specId}} link actually causes it to be created though, or if it is just convention that there will be a spec fo each environment.

Probably something we should understand.

Reply:
The manifest-template.yaml is configured to create manifest for Alpha, RC and FIX. But PR build only deploy Alpha manifest and skips RC and FIX.  The RC and FIX deployments were removed from [azure/azure-pr-pipeline.yml](https://github.com/NHSDigital/e-referrals-service-api/commit/21a7f0020f1976ad44cd6f55ae5ce5d4703e5ab8#diff-d83453f08d4bb08444786b2940fc90a58a3f245bc5c3a5f65070f3e702623719) by Hubert using ERSSUP-55776. After the investigation, Hubert did this as temporary fix until find the proper solution.  

Spoke Jordan who tested ERSSUP-55776 saying that Hubert confirmed that FIX and RC were no longer required for PRs. 
![Hubert-Slack](https://user-images.githubusercontent.com/87030441/144600660-ec652362-9f41-4804-8407-e40281f37140.png)

We have to investigate via a bug if we need FIX and RC in PR builds.